sdk/monitor/opentelemetry-exporter-azure-monitor/src/utils/connectionStringParser.ts

```diff
@@ -69,4 +72,17 @@ export class ConnectionStringParser {
 
     return result;
   }
+
+  public static sanitizeUrl(url: string) {
+    let newUrl = url.trim();
+    if (newUrl.indexOf("https") < 0) {
```

Comment:
NIT: To be more specific, you really want to match `https://`, right?

And by the same token, you want to replace `http://`. This would prevent (maybe unlikely) replacement of the text if `http` or `https` appeared in the URL. I'm not sure if any part of your URL is randomly generated or user chosen, for instance.